demo_nodes_cpp/src/parameters/ros2param.cpp

```diff
@@ -139,6 +139,12 @@ int main(int argc, char ** argv)
       fprintf(stderr, "Failed to set parameter\n");
       return 1;
     }
+    auto results = set_parameters_result.get();
+    for (auto result : results) {
+      if (!result.successful) {
+        fprintf(stderr, "Error setting parameter: %s\n", result.reason.c_str());
```

Comment:
The command line interface only supports one parameter at a time, but there is no API for setting one parameter.